xpk.py

```diff
@@ -2553,7 +3161,11 @@ def workload_delete(args) -> int:
       args.workload = workload
       yml_string = workload_delete_yaml.format(args=args)
       tmp = write_temporary_file(yml_string)
-      command = f'kubectl delete -f {str(tmp.file.name)}'
+      device_type = args.tpu_type if args.tpu_type else args.device_type
+      if device_type == h100_device_type:
+        command = f'kubectl delete jobset {workload} -n default'
```

Comment:
Thanks Nina, I think we don't even need to parse if it is a TPU / CPU / GPU.

Can we use `kubectl delete jobset {workload} -n default` for all delete cases and delete the --tpu-type and --device-type required arguments for the workload_delete_parser?

This allows us to avoid creating new required arguments, and delete the not needed workload_delete_yaml (big yay).